doc/build-unix.md

```diff
@@ -195,19 +189,19 @@ Additional Configure Flags
 --------------------------
 A list of additional configure flags can be displayed with:
 
-    ./configure --help
+    cmake -B build -LH
```

Comment:
Just a suggestion: We could also mention `ccache` here.

Reply:
Mind elaborating your suggestion?

FWIW, there is _"Cache compilations with `ccache`"_ section in https://github.com/bitcoin/bitcoin/blob/master/doc/productivity.md.

Reply:
Just an unrelated passing comment. I guess it makes more sense in productivity.md. Please resolve.